packages/patternfly-4/react-core/src/components/Select/Select.d.ts

```diff
@@ -0,0 +1,16 @@
+import { HTMLProps, FormEvent } from 'react';
+import { Omit } from '../../typeUtils';
+
+export interface SelectProps extends Omit<HTMLProps<HTMLOptionElement>> {
+  isExpanded?: boolean;
+  onSelect: Function;
+  onToggle: Function;
+  placeholderText?: string;
+  selectOptions?: array;
```

Comment:
Array of what? to not type it (which shouldn't be done) you would want to use `any[]` or `Array<any>`